index.js

```diff
@@ -27,7 +27,7 @@ document.addEventListener('DOMContentLoaded', () => {    fetch('http://localhost
 
```

Comment:
- Line 2 Hit enter after the { (`fetch` should be on a new line)
- Delete line 5 and other empty lines
- Line 7 change variable name to "searchForm" or similar